apps/test/unit/gamelab/animationListModuleTest.js

```diff
@@ -563,4 +564,34 @@ describe('animationListModule', function () {
       expect(store.getState().animationList.pendingFrames.props).to.deep.equal(libraryAnimProps);
     });
   });
+
+  describe('action: save animation', function () {
+    beforeEach(function () {
+      this.xhr = sinon.useFakeXMLHttpRequest();
+      let requests = this.requests = [];
+      this.xhr.onCreate = function (xhr) {
```

Comment:
fair, it's fuzzy to me too. I probably relied too heavily on the documentation example code here.